examples/basic/main.tf

```diff
@@ -41,7 +41,7 @@ module "vector_log_cloudwatch" {
   cluster_identity_oidc_issuer     = module.eks_cluster.eks_cluster_identity_oidc_issuer
   cluster_identity_oidc_issuer_arn = module.eks_cluster.eks_cluster_identity_oidc_issuer_arn
 
-  settings = {
+  helm_set = {
```

Comment:
update input variables, and create an example testing matrix that is used for dummy testing with terraform plan in pipelines. It would nice to have test on all installation paths: helm, helm+argo, argo with cw and/or opensearch sinks.

Reply:
> PS: to be future proof we can add irsa_additional_policies parameter to the IAM section

irsa_additional_policies added into iam.tf

Reply:
tests added